### ISSUE TITLE
Submitting to multiple recipients and returned message ID

### ISSUE THREAD

**Issue by faxal** [fiorix/go-smpp#72]
From the SMSC I'm connecting to (not sure which SMSC) when submitting to multiple recipients Response Message contains a single ID, but when delivery report is received it has different message IDs for each recipient. 

Is there a way to retrieve message ID for each recipient?

**Comment by faxal** [comment]
According to the spec ( as far as I understand ) this is not possible, as only a single ID is returned in the response. 

So if returned message ID is important ( for DLR etc. ) only option is to avoid batch message sends and send them one by one.

**Comment by faxal** [comment]
@McSwitch That is correct. What I am asking may not be possible from what I have gathered.

**Comment by fiorix** [comment]
Closing as a support/usage question. For usage help, please refer to the GoDoc and examples in the repository.
